lib/DebugTrackWastedRenders.spec.lua

```diff
@@ -0,0 +1,8 @@
+return function()
+
+	describe("wastedRenderTracking", function()
+		it("should print each time a set of renders completes", function ()
+			-- TODO: Figure out how to test this given that it connects to heartbeat
```

Comment:
This will probably require knowledge of the internal state of the tracker; I think it's okay to expose that, since we're not exporting the module in `init.lua`! With that said, testing this right now is not possible, since lemur doesn't have an event loop for events to use.